### PR TITLE
Try to build and develop locally

### PR DESCRIPTION
**Pull request by skyl**
allow flutter root git in Dockerfile, ios output/lock

### CODE REVIEW

**Comment by skyl** [comment]
this is just a draft as I try to get it running/building locally...

File: ios/Podfile.lock [hunk 1]
```diff
@@ -0,0 +1,193 @@
+PODS:
```
Comment (skyl) [inline ios/Podfile.lock:1]: I think you probably do want to check in the lock file?

**Comment by hhanh00** [comment]
It's created by the flutter build.

**Comment by hhanh00** [comment]
This PR was closed because it has been stalled with no activity.

**Pull request closed without merging**